examples/webgl_lights_spotlight.html

```diff
@@ -232,6 +233,15 @@
 
 				} );
 
+
+				gui.add( params, 'enable shadows' ).onChange( function ( val ) {
+
+					renderer.shadowMap.enabled = val
```

Comment:
Semicolon is missing here.

Reply:
Fixed. 5ae10928d04adc176d17e57f3782fad7eb8f7a94